### PR TITLE
Booster vaccination UI (EXPOSUREAPP-9041)

### PR DESCRIPTION
**Pull request by mtwalli**
### Changes 
- Add label for booster vaccination
- Adapted screenshots to reflect new changes 
- Updated dialog text

### Testing 
- scan `cwa vc gen -s v.0.dn=1 v.0.sd=2 -e wru --seed mm`
- scan `cwa vc gen -s v.0.dn=2 v.0.sd=2 -e wru --seed mm`
- scan `cwa vc gen -s v.0.dn=3 v.0.sd=2 -e wru --seed mm` (Booster)

### CODE REVIEW

File: Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <!-- XTXT: Vaccination List deletion dialog title-->
     <string name="vaccination_list_deletion_dialog_title">Wollen Sie das Impfzertifikat wirklich entfernen?</string>
     <!-- XTXT: Vaccination List deletion dialog message-->
-    <string name="vaccination_list_deletion_dialog_message">Wenn Sie das Impfzertifikat entfernen, kann die App die Impfung nicht mehr für die Prüfung Ihres Impfstatus berücksichtigen.</string>
+    <string name="vaccination_list_deletion_dialog_message">Wir empfehlen die Impfzertifikate in der App zu behalten, auch wenn Sie diese nicht mehr aktuell sind.</string>
```
Comment (InkenTownsend) [inline Corona-Warn-App/src/main/res/values-de/vaccination_strings.xml:13]:
```suggestion
    <string name="vaccination_list_deletion_dialog_message">Wir empfehlen die Impfzertifikate in der App zu behalten, auch wenn diese nicht mehr aktuell sind.</string>
```

**Comment by InkenTownsend** [comment]
@mtwalli there was one word too many that I removed in my comment.

**Comment by mtwalli** [comment]
> there was one word too many that I removed in my comment.

@InkenTownsend  done

**Review by InkenTownsend — APPROVED** [review]
UA approved

**Review by jurajkusnier — COMMENTED** [review]
Is this expected behavior?

https://user-images.githubusercontent.com/2919625/130622674-fea41ffd-3295-4036-a6c7-8dde92f48719.mp4

**Comment by mtwalli** [comment]
> Is this expected behavior?
> 
>  device-2021-08-24-151239.mp4

I think this is happening because all vaccination certificates have the same `vaccinatedOn` date, can you try to change the `vaccinatedOn` per certificate? maybe using this webpage https://dgc.a-sit.at/ehn/ is easier to play with the date

**Review by jurajkusnier — APPROVED** [review]

**Review by BMItr — APPROVED** [review]
code lgtm.
test on OnePlus 8 successful.

image

**Pull request merged**